pkg/csi/service/common/common_controller_helper.go

```diff
@@ -189,7 +189,7 @@ func IsOnlineExpansion(ctx context.Context, volumeID string, nodes []*cnsvsphere
 		log.Error(msg)
 		return status.Errorf(codes.Internal, msg)
 	} else if diskUUID != "" {
-		msg := fmt.Sprintf("failed to expand volume: %q. Volume is attached to node. Only offline volume expansion is supported", volumeID)
+		msg := fmt.Sprintf("failed to expand volume: %q. Volume is attached to node. Online volume expansion is not supported in this version", volumeID)
```

Comment:
`IsOnlineExpansion` should return true in this case instead of returning error.

Reply:
This function is only run in scenarios where online resize is not enabled or supported. Therefore, we need to error out.

Reply:
The method signature seems incorrect. If this method is called `IsOnlineExpansion`, it should return true or false with error if any.

Reply:
Raised an internal bug and assigned it to myself.